app/moves/middleware.js

```diff
@@ -81,6 +96,33 @@ module.exports = {
 
     next()
   },
+  setMoveTypeNavigation: async (req, res, next) => {
+    const { period, dateRange } = res.locals
```

Comment:
You can also get `period` from `req.params`? Removes the dependency on order that way.